javascript-operators-and-expressions/main.js

```diff
@@ -0,0 +1,46 @@
+const width = 7;
+const height = 4;
+const area = width * height;
+
+console.log('value of area: ', area);
+console.log('typeof area: ', typeof area);
+
+const bill = 20;
+const payment = 10;
+const change = bill - payment;
```

Comment:
Your `payment` should be larger than the `bill`, and they will need to be swapped on line 10. You need to pay enough to cover your tab right? 😉